EIPS/eip-4635.md

```diff
@@ -0,0 +1,312 @@
+    - It MUST return the constant value `true` if `0xba29fa60` is passed through the `interfaceID` argument. This signifies ERC-SFT `IERC4635Receiver` support.
+
+### Metadata
+```solidity
+
+pragma solidity ^0.8.0;
+
+interface IERC4635Metadata /* is IERC4635 */{
+    /**
+        @notice A descriptive name for a collection of SFTs in this contract
+    */
+    function semiName(uint256 _tokenType) external view returns (string memory);
+    /**
+        @notice An abbreviated name for SFTs in this contract
+    */
+    function semiSymbol(uint256 _tokenType) external view returns (string memory);
+    /**
+        @notice A distinct Uniform Resource Identifier (URI) for a given asset.
+        @dev Throws if `_id` is not a valid NFT. URIs are defined in RFC
+        3986. The URI may point to a JSON file that conforms to the "ERC721
+        Metadata JSON Schema".
+    */
+    function semiURI(uint256 _tokenType) external view returns (string memory);
+}
+```
+
+## Rationale
+The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages.
+
+## Backwards Compatibility
+All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.
+
+## Test Cases
+Test cases for an implementation are mandatory for EIPs that are affecting consensus changes.  If the test suite is too large to reasonably be included inline, then consider adding it as one or more files in `../assets/eip-####/`.
+
+## Reference
+**Standards**
+- [ERC-721 Non-Fungible Token Standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md)
+- [ERC-165 Standard Interface Detection](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md)
+- [ERC-1155 Multi Token Standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1155.md)
+
+**Implementations**
+- [ERC-SFT Reference Implementation](https://github.com/linvictor88/ERC-SFT)
+
+## Security Considerations
+All EIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. EIP submissions missing the "Security Considerations" section will be rejected. An EIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.
```

Comment:
Please remove this text and describe any security considerations regarding the EIP.

Reply:
OK. Thanks.